README.md

```diff
@@ -114,16 +146,38 @@ var styles = StyleSheet.create({
 module.exports = Launch;
 ```
 
-## Getting started
-1. `npm install react-native-router-flux --save`
-2. In top-level index.js:
-    * Define Route for each app screen. Its 'type' attribute is 'push' by default, but you also could define 'replace', so navigator will replace current route with new route.
-    'switch' type represents tab screen. 'component' attribute is React component class which will be created for this route and all route attributes will be passed to it.
-Instead of defining 'component' class you could define any child (one) you want to be used for that route (even one more Router)
-'wrapRouter' - adds Router child for this Route (so defines own navigator for the route). 'name' is unique name of Route.
-All other attributes will be passed to scene class.
-
-    * If some your Routes have common attributes, you may define Schema element and just use 'schema' attribute for 'route'
-3. In any app screen:
-    * var {Actions} = require('react-native-router-flux');
-    * Actions.ACTION_NAME(PARAMS) will call appropriate action and params will be passed to the route
+## Redux/Flux support
+This component is not opinionated and does not depend on any implementation of Flux or Redux.
+
+All route actions can be hooked by adding handlers for `Actions.onPush`, `Actions.onReplace`, `Actions.onPop` in your store(s).
+
+If a handler returns false route action is ignored. For Redux, you will need to 'connect' your component to your store.
```

Comment:
sorry that was more of a grammatical comment. I think the word `the` should be inserted in that sentence is all